link.py

```diff
@@ -5,14 +5,13 @@ class Link(object):
        
     Attributes:
         ns (NetworkSimulator): stores the simulator class running the simulation
-        link_id (int): a unique identifier for each link
+        link_id (string): a unique identifier for each link
         max_buffer_size (int): how many bits can be stored in the link buffer
```

Comment:
So in the flow stuff we're making everything ints (even bits).  Idk if you want to do that here for consistency?